AdyenActions/UI/View Controllers/DelegatedAuthentication/DelegatedAuthenticationView.swift

```diff
@@ -0,0 +1,208 @@
+        self.headerTextStyle = headerTextStyle
+        self.descriptionTextStyle = descriptionTextStyle
+        self.progressViewStyle = progressViewStyle
+        self.progressTextStyle = progressTextStyle
+        self.firstButtonStyle = firstButtonStyle
+        self.secondButtonStyle = secondButtonStyle
+        self.textViewStyle = textViewStyle
+
+        super.init(frame: .zero)
+        configureViews()
+    }
+    
+    @available(*, unavailable)
+    internal required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Configuration
+        
+    private func configureViews() {
+        addSubview(image)
+        addSubview(labelsStackView)
+        addSubview(progressStackView)
+        addSubview(buttonsStackView)
+        addSubview(textView)
+
+        NSLayoutConstraint.activate([
+            image.topAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor, constant: 50),
+            image.centerXAnchor.constraint(equalTo: layoutMarginsGuide.centerXAnchor),
+            image.widthAnchor.constraint(equalToConstant: 30),
+            image.heightAnchor.constraint(equalToConstant: 34),
+
+            labelsStackView.topAnchor.constraint(equalTo: image.bottomAnchor, constant: 24),
+            labelsStackView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            labelsStackView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor, constant: 15.0),
+            labelsStackView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor, constant: -15.0),
+
+            progressView.widthAnchor.constraint(equalToConstant: 200),
+            progressStackView.topAnchor.constraint(equalTo: labelsStackView.bottomAnchor, constant: 24),
+            progressStackView.centerXAnchor.constraint(equalTo: layoutMarginsGuide.centerXAnchor),
+
+            firstButton.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor),
+            firstButton.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor),
+
+            secondButton.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor),
+            secondButton.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor),
+
+            buttonsStackView.topAnchor.constraint(equalTo: progressStackView.bottomAnchor, constant: 24),
+            buttonsStackView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor),
+            buttonsStackView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor),
+            
+            textView.heightAnchor.constraint(equalToConstant: 50),
+            textView.topAnchor.constraint(equalTo: buttonsStackView.bottomAnchor, constant: 24),
+            textView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor, constant: -15),
+            textView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor, constant: 15.0)
+        ])
```

Comment:
How does it look on iPad and landscape ?

Reply:
It pins the content to the top similar to the card view. So i would think it would be ok. In Drop in there is a lot of space at the bottom, i'll check if the view can be made smaller there. It is controlled by the preferredContentSize, which doesn't seem to get a correct height.

Reply:
Got it. Corrected the constraints a bit. 

![Simulator Screenshot - iPad Pro (12 9-inch) (4th generation) - 2023-06-13 at 15 43 54](https://github.com/Adyen/adyen-ios/assets/96450350/3ac2df0b-bf7a-40cc-b8a9-645b8c74368a)